src/ui/components/map/mapcomponent.js

```diff
@@ -81,7 +81,9 @@ export default class MapComponent extends Component {
     }
 
     if (data.resultsContext === ResultsContext.NO_RESULTS && !this._noResults.displayAllResults) {
```

Comment:
Does this conditional need to be updated to check _noResults.visible as well?

Reply:
asking Rose, I interpreted this as noResults.visible only controls whether an empty map will show up, and will not affect whether map pins will show up or not (which is what displayAllResults does). I think I likely misunderstood things.

Reply:
> so noResults.visible is true, show the map on no results. if
> noResults.displayAllResults is also true, show the pins on the map

Ok apparently I didn't misread things

Reply:
In other words, no this conditional should not be updated.